torchci/test/mergeCommands.test.ts

```diff
@@ -125,7 +124,7 @@ describe("merge-bot", () => {
       .reply(200, {})
       .post(`/repos/${owner}/${repo}/dispatches`, (body) => {
         expect(JSON.stringify(body)).toContain(
-          `{"event_type":"try-merge","client_payload":{"pr_num":${pr_number},"comment_id":${comment_number},"force":true}}`
+          `{"event_type":"try-merge","client_payload":{"pr_num":${pr_number},"comment_id":${comment_number},"force":true,"land_checks":true}}`
```

Comment:
This is a bit confusing--can force AND landchecks be true at the same time?

Reply:
That shouldn't be possible when using merge bot given that these options are mutually exclusive.  May be `land_checks: landChecks || landChecksEnrolled` on https://github.com/pytorch/test-infra/blob/main/torchci/lib/bot/pytorchBotHandler.ts#L181 should just be `land_checks: landChecks` because `landChecksEnrolled` is always true?

Reply:
Yes it's confusing. And yes this has always been the case in the sense that we've been sending this value for everyone who's been opted in for land checks

The actual behavior in this scenario is that the `force:true` takes precedence and land checks never get run

Reply:
Here's where we skip land checks if force was set (`skip_mandatory_checks` == force):

https://github.com/pytorch/pytorch/blob/master/.github/scripts/trymerge.py#L1338